Suko/Model/SUKAnime.h

```diff
@@ -20,13 +20,13 @@ NS_ASSUME_NONNULL_BEGIN
 /** The anime synopsis */
 @property (nonatomic) NSString *synopsis;
 /** The genres this anime fit into */
-@property (nonatomic) NSArray *genres;
+@property (nonatomic) NSArray<NSDictionary*> *genres;
 /** Episode count */
 @property (nonatomic) int episodes;
```

Comment:
nit: Would name this variable "numEpisodes" to signify it's a count variable rather than a list of objects